maestro/docker-compose.dev.yml

```diff
@@ -0,0 +1,50 @@
+version: '3'
```

Comment:
is this file be used anywhere?

Reply:
this file is to test this chart locally, it's not part of the chart directory